apps/openmw/mwphysics/physicssystem.cpp

```diff
@@ -1152,9 +1152,9 @@ namespace MWPhysics
 
                 bool waterCollision = false;
                 if (effects.get(ESM::MagicEffect::WaterWalking).getMagnitude()
-                        && cell->getCell()->hasWater()
-                        && !world->isUnderwater(iter->first.getCell(),
-                                               osg::Vec3f(iter->first.getRefData().getPosition().asVec3())))
+                        && cell->getCell()->hasWater())
+//                        && !world->isUnderwater(iter->first.getCell(),
+//                                               osg::Vec3f(iter->first.getRefData().getPosition().asVec3())))
```

Comment:
You know that it would take you only a few seconds to remove these lines by
yourselves ?
For me, each time I go back to this branch it means recompiling a lot of
stuff when I go back to my normal branch, it's just removing instead of
commenting out, a pure loss of time for me, you are welcome to do it
anytime you want !
Anyway, I guess I'll do it later, just to be sure it's ok now !

2015-10-01 21:59 GMT+02:00 Bret Curtis <notifications@github.com>:

> In apps/openmw/mwphysics/physicssystem.cpp
> <https://github.com/OpenMW/openmw/pull/761#discussion_r40958963>:
>
> > @@ -1152,9 +1152,9 @@ namespace MWPhysics
> >
> >                  bool waterCollision = false;
> >                  if (effects.get(ESM::MagicEffect::WaterWalking).getMagnitude()
> > -                        && cell->getCell()->hasWater()
> > -                        && !world->isUnderwater(iter->first.getCell(),
> > -                                               osg::Vec3f(iter->first.getRefData().getPosition().asVec3())))
> > +                        && cell->getCell()->hasWater())
> > +//                        && !world->isUnderwater(iter->first.getCell(),
> > +//                                               osg::Vec3f(iter->first.getRefData().getPosition().asVec3())))
>
> Please do what scrawl suggests, we can always go back in git history to
> get it if we needed it again.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenMW/openmw/pull/761/files#r40958963>.
>